wsi/drm/drmdisplaymanager.cpp

```diff
@@ -585,7 +585,7 @@ NativeDisplay *DrmDisplayManager::CreateVirtualPanoramaDisplay(
   NativeDisplay *latest_display;
   std::unique_ptr<VirtualPanoramaDisplay> display(
       new VirtualPanoramaDisplay(fd_, buffer_handler_.get(), display_index, 0));
-  virtual_displays_.emplace_back(std::move(display));
+  virtual_displays_.emplace(display_index,std::move(display));
```

Comment:
The returned VirtualPanoramaDisplay is not the one just created, this will introduce the HWC crash. Actually, there is no need to put new created VirtualPanoramaDisplay into map as it will introduce another crash when surfaceflinger handle the hotplug. You could refer the commit https://github.com/intel/IA-Hardware-Composer/pull/550, just create the the instance and return it accordingly.

Reply:
update it with your patch on acrn stable